lib/assembly/broadway.ex

```diff
@@ -29,6 +30,17 @@ defmodule Assembly.Broadway do
   @impl true
   @decorate transaction(:queue)
   def handle_message(_, %Message{} = message, _context) do
+    bottle =
+      message
+      |> URI.decode()
+      |> Bottle.Core.V1.Bottle.decode()
+
+    Bottle.RequestId.read(:queue, bottle)
+
+    with {:error, reason} <- notify_handler(bottle.resource) do
```

Comment:
Don't we need to send the event here as well to match the function head below

Reply:
The resource is itself a tuple: https://github.com/system76/bottle/blob/elixir/lib/bottle/core/v1/bottle.pb.ex#L6